robot/Crisis-Management/resources/ContactPageObject.py

```diff
@@ -24,14 +24,35 @@ def login_to_community_as_user(self):
         landing_tab=cm_lex_locators["link-text"].format("Home")
         status=self.selenium.get_webelement(landing_tab).get_attribute("class")
         assert "active" in status, "Community landing tab is not Home"
-        
+		
     def _is_current_page(self):
         """ Verify we are on the Contact detail page
             by verifying that the url contains '/view'
         """
         self.selenium.wait_until_location_contains("/view", timeout=60, message="Detail page did not load in 1 min")
         self.selenium.location_should_contain("/lightning/r/Contact/",message="Current page is not a Contact record detail view")
         
+    def navigate_and_verify_availability_related_list(self, name, expected_count):
+        """Delete the old value in specified field by clicking on delete icon and update with new value"""
+        base_url = self.cumulusci.org.lightning_base_url
+        contact_id = self.salesforce.get_current_record_id()
+        url = "{}/lightning/r/{}/related/Assignments__r/view".format(base_url,contact_id)
```

Comment:
@rjanjanam I'm wondering if there is any reason to make this specific to the Assignment object and not generic so that you can feed in any object. Since there are related lists on every page, it seems it could be used again in the future to count any related object.

Besides that, this looks good to me!

Reply:
agree with Julian

Reply:
This is slightly different not the regular related tab. I felt this is relevant and serves a specific purpose. Happy to explain.